src/LogicTasks/Semantics/Fill.hs

```diff
@@ -152,26 +162,30 @@ partialGrade FillInst{..} sol = do
       solLen = length boolSol
       missingLen = length missing
 
-completeGrade :: OutputCapable m => FillInst -> [TruthValue] -> LangM m
-completeGrade FillInst{..} sol = do
-  preventWithHint (notNull diff)
-    (translate $ do
-      german "Lösung ist korrekt?"
-      english "Solution is correct?"
-    )
-    (do
-      translate $ do
-        german $ "Die Lösung beinhaltet " ++ displayMistake ++ " Fehler."
-        english $ "Your solution contains " ++ displayMistake ++ " mistakes."
-      when showSolution $ example (show missingValues) $ do
-        english "The solution for this task is:"
-        german "Die Lösung für die Aufgabe ist:"
-      pure ()
-    )
+completeGrade :: (OutputCapable m, Alternative m, Monad m) => FillInst -> [TruthValue] -> Rated m
+completeGrade FillInst{..} sol = reRefuse
+  (extendedMultipleChoice
+    (MinimumThreshold (1 % 2))
+    (Punishment 0)
+    (TargetedCorrect (length solution))
```

Comment:
This looks a lot like https://github.com/fmidue/output-blocks/blob/cc5e88053b905827221a736355b6c0678c6ed6cd/output-blocks/src/Control/OutputCapable/Blocks.hs#L204-L208 (same parameters). So, is it possible to use `multipleChoice` instead of `extendedMultipleChoice` here?

Reply:
> So, is it possible to use multipleChoice instead of extendedMultipleChoice here?

No. The simple `multipleChoice` is not suitable for this task. Especially `[a]` vs `Map a Bool` (type of the submission) matters here.